packages/garbo/src/config.ts

```diff
@@ -83,12 +83,16 @@ function stringToWorkshedItem(s: string): Item | null {
   return validWorksheds[0].item;
 }
 
+// We want this to be a const because we don't want to determine we can no longer target Cheerless Mime Executive simply because we used all reminisces to start a chain
+const locketAvailable = CombatLoversLocket.reminiscesLeft() > 0;
+
 function defaultTarget() {
   if ($skills`Curse of Weaksauce, Saucegeyser`.every((s) => have(s))) {
     if (
       ChestMimic.have() ||
-      have($item`Clan VIP Lounge key`) ||
+      (have($item`Clan VIP Lounge key`) && !get("_photocopyUsed")) ||
       (CombatLoversLocket.have() &&
+        locketAvailable &&
```

Comment:
my only small caveat is that this doesn't quite handle edge cases -- i believe the locket monster pref doesn't remove monsters you've fought today, so this could be improved by making the locketAvailable logic also check that you have not reminisced the desired target today. but this + the check on photocopy usage seem straightforward and helpful.

Reply:
Locket does track the monsters reminisced, so I've included a check for this. Again, as part of the const so that reminiscing doesn't somehow change our target retroactively.